PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -3538,13 +3543,13 @@ struct HfTrackIndexSkimCreatorLfCascades {
             if (fillHistograms) {
               registry.fill(HIST("hFitterStatusXi2Prong"), 1);
             }
-            continue;
+            isGoogForXi2Prong = false;
           }
-          if (fillHistograms) {
+          if (isGoogForXi2Prong && fillHistograms) {
             registry.fill(HIST("hFitterStatusXi2Prong"), 0);
           }
 
-          if (nVtxFrom2ProngFitterXiHyp > 0) {
+          if (isGoogForXi2Prong && nVtxFrom2ProngFitterXiHyp > 0) {
```

Comment:
`nVtxFrom2ProngFitterXiHyp` should be `0` if the `DCAFitter` throws the exception, so the `isGoodForXi2Prong` is redundant (it is sufficient to remove the `continue` above)